Pod/Classes/KolodaView/KolodaView.swift

```diff
@@ -657,7 +657,7 @@ public class KolodaView: UIView, DraggableCardDelegate {
         if !visibleIndexes.isEmpty {
             proceedDeletionInRange(visibleIndexes[0]...visibleIndexes[visibleIndexes.count - 1])
         }
-        currentCardIndex -= Array(indexRange).filter { $0 < currentCardIndex }.count
+        currentCardIndex += Array(indexRange).filter { $0 < currentCardIndex }.count
```

Comment:
Let me explain. Cards can be swiped programatically only with `.swipe(.direction)` method. 
In this case you want to delete unused cards both from Koloda and your data source, so they are synced.
For example, here is your data source:` [orangeCard, yellowCard, redCard, blueCard, greenCard]`
`currentItemIndex == 2 ` - so redCard is on top.
you remove orangeCard, yellowCard from your data source and you call Koloda `remove` method at indexes 0...1 .
Now your data source state is :  ` [redCard, blueCard, greenCard]`
So redCard is still on top and Koloda has `currentItemIndex == 0 `